<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="GET Feeds - Duplicate Transactions"
           tests="16"
           failures="12"
           errors="0"
           skipped="0"
           time="272.811">
  <properties />
            <testcase name="testGetNotificationsForDuplicateTransaction" time="16.378" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  com.yodlee.insights.yodleeApi.utils.InsightsGenerics.getExpectedAttributesToBeValidatedV2(InsightsGenerics.java:1326)
  com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:257)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
  java.lang.Thread.run(Unknown Source)
]]>
      </failure>

          </testcase>
            <testcase name="testGetNotificationsForDuplicateTransaction" time="19.152" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
                <failure type="junit.framework.AssertionFailedError"
                  message="Expected  2 insights triggered But found Empty {}"
              >
<![CDATA[junit.framework.AssertionFailedError: Expected  2 insights triggered But found Empty {}
  junit.framework.Assert.fail(Assert.java:57)
  com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:307)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
  java.lang.Thread.run(Unknown Source)
]]>
      </failure>

          </testcase>
            <testcase name="testGetNotificationsForDuplicateTransaction" time="16.913" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
                <failure type="junit.framework.AssertionFailedError"
                  message="Expected  2 insights triggered But found Empty {}"
              >
<![CDATA[junit.framework.AssertionFailedError: Expected  2 insights triggered But found Empty {}
  junit.framework.Assert.fail(Assert.java:57)
  com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:307)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
  java.lang.Thread.run(Unknown Source)
]]>
      </failure>

          </testcase>
            <testcase name="testGetNotificationsForDuplicateTransaction" time="16.738" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
                <failure type="junit.framework.AssertionFailedError"
                  message="Expected  2 insights triggered But found Empty {}"
              >
<![CDATA[junit.framework.AssertionFailedError: Expected  2 insights triggered But found Empty {}
  junit.framework.Assert.fail(Assert.java:57)
  com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:307)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
  java.lang.Thread.run(Unknown Source)
]]>
      </failure>

          </testcase>
            <testcase name="testGetNotificationsForDuplicateTransaction" time="15.998" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  com.yodlee.insights.yodleeApi.utils.InsightsGenerics.getExpectedAttributesToBeValidatedV2(InsightsGenerics.java:1326)
  com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:257)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
  java.lang.Thread.run(Unknown Source)
]]>
      </failure>

          </testcase>
            <testcase name="testGetNotificationsForDuplicateTransaction" time="16.503" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
                <failure type="junit.framework.AssertionFailedError"
                  message="Expected  2 insights triggered But found Empty {}"
              >
<![CDATA[junit.framework.AssertionFailedError: Expected  2 insights triggered But found Empty {}
  junit.framework.Assert.fail(Assert.java:57)
  com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:307)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
  java.lang.Thread.run(Unknown Source)
]]>
      </failure>

          </testcase>
            <testcase name="testGetNotificationsForDuplicateTransaction" time="16.865" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
                <failure type="junit.framework.AssertionFailedError"
                  message="Expected  2 insights triggered But found Empty {}"
              >
<![CDATA[junit.framework.AssertionFailedError: Expected  2 insights triggered But found Empty {}
  junit.framework.Assert.fail(Assert.java:57)
  com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:307)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
  java.lang.Thread.run(Unknown Source)
]]>
      </failure>

          </testcase>
            <testcase name="testGetNotificationsForDuplicateTransaction" time="16.708" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
                <failure type="junit.framework.AssertionFailedError"
                  message="Expected  2 insights triggered But found Empty {}"
              >
<![CDATA[junit.framework.AssertionFailedError: Expected  2 insights triggered But found Empty {}
  junit.framework.Assert.fail(Assert.java:57)
  com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:307)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
  java.lang.Thread.run(Unknown Source)
]]>
      </failure>

          </testcase>
            <testcase name="testGetNotificationsForDuplicateTransaction" time="16.444" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
                <failure type="junit.framework.AssertionFailedError"
                  message="Expected  2 insights triggered But found Empty {}"
              >
<![CDATA[junit.framework.AssertionFailedError: Expected  2 insights triggered But found Empty {}
  junit.framework.Assert.fail(Assert.java:57)
  com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:307)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
  java.lang.Thread.run(Unknown Source)
]]>
      </failure>

          </testcase>
            <testcase name="testGetNotificationsForDuplicateTransaction" time="16.933" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
                <failure type="junit.framework.AssertionFailedError"
                  message="Expected  2 insights triggered But found Empty {}"
              >
<![CDATA[junit.framework.AssertionFailedError: Expected  2 insights triggered But found Empty {}
  junit.framework.Assert.fail(Assert.java:57)
  com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:307)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
  java.lang.Thread.run(Unknown Source)
]]>
      </failure>

          </testcase>
            <testcase name="testGetNotificationsForDuplicateTransaction" time="16.643" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  com.yodlee.insights.yodleeApi.utils.InsightsGenerics.getExpectedAttributesToBeValidatedV2(InsightsGenerics.java:1326)
  com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:257)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
  java.lang.Thread.run(Unknown Source)
]]>
      </failure>

          </testcase>
            <testcase name="testGetNotificationsForDuplicateTransaction" time="17.154" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions">
                <failure type="junit.framework.AssertionFailedError"
                  message="Expected  2 insights triggered But found Empty {}"
              >
<![CDATA[junit.framework.AssertionFailedError: Expected  2 insights triggered But found Empty {}
  junit.framework.Assert.fail(Assert.java:57)
  com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions.testGetNotificationsForDuplicateTransaction(TestGetInsightsForDuplicateTransactions.java:307)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
  java.lang.Thread.run(Unknown Source)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="testGetNotificationsForDuplicateTransaction" time="17.925" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions" />
                <testcase name="testGetNotificationsForDuplicateTransaction" time="18.502" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions" />
                <testcase name="testGetNotificationsForDuplicateTransaction" time="16.157" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions" />
                <testcase name="testGetNotificationsForDuplicateTransaction" time="17.798" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForDuplicateTransactions" />
      
</testsuite>
